src/components/ReportActionItem/RenameAction.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+
+const propTypes = {
+    /** All the data of the action */
+    action: PropTypes.shape(reportActionPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const RenameAction = (props) => {
+    const displayName = lodashGet(props.action, ['message', 0, 'text']);
+    const oldName = lodashGet(props.action, 'originalMessage.oldName', '');
+    const newName = lodashGet(props.action, 'originalMessage.newName', '');
+
+    return (
+        <Text style={[styles.pv2, styles.ph5, styles.textAlignCenter, styles.textLabelSupporting]}>
+            <Text style={[styles.textLabelSupporting, styles.textStrong]}>
```

Comment:
Since this is already wrapped in `textLabelSupporting` above, do you need it again here?

Reply:
That was my initial implementation and given the example given in the [RN docs](https://reactnative.dev/docs/text#limited-style-inheritance), it should have worked without `textLabelSupporting` in the nested `Text`. However, when I remove it from the nested text, the font color and size are incorrect 🤷

Reply:
Ah got it, thanks for clarifying!